### PR TITLE
Change hide! to the implementation to support rails 3.2

### PR DESCRIPTION
**Pull request by flynfish**
This allows the generators to work in rails 3.2 because the hide! method only exists in rails 4.

### CODE REVIEW

File: lib/generators/react_on_rails/base_generator.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 module ReactOnRails
   module Generators
     class BaseGenerator < Rails::Generators::Base # rubocop:disable Metrics/ClassLength
-      hide!
+      Rails::Generators.hide_namespace self.namespace
```
Comment (justin808) [inline lib/generators/react_on_rails/base_generator.rb]: Let's use parens for the method call.

Any doc link for this?

Comment (flynfish) [inline lib/generators/react_on_rails/base_generator.rb]: Is this what you are looking for?
http://apidock.com/rails/Rails/Generators/hide_namespace/class

Comment (justin808) [inline lib/generators/react_on_rails/base_generator.rb]: Yep! In both 4.2.1 and 3.2.13!

http://apidock.com/rails/v3.2.13/Rails/Generators/hide_namespace/class

http://apidock.com/rails/v4.2.1/Rails/Generators/hide_namespace/class

**Comment by justin808** [comment]
Please make the paren change, and I'll merge.

**Comment by flynfish** [comment]
@justin808 done

**Pull request merged**